airbyte-cdk/python/airbyte_cdk/sources/streams/concurrent/adapters.py

```diff
@@ -166,8 +194,7 @@ def get_json_schema(self) -> Mapping[str, Any]:
 
     @property
     def supports_incremental(self) -> bool:
-        # Only full refresh is supported
-        return False
+        return not isinstance(self._cursor, NoopCursor)
```

Comment:
Eventually, we should be good not exposing `supports_incremental` but in the meanwhile, we rely on knowing the implementation detail to support the previous interface